.github/labeler.yml

```diff
@@ -0,0 +1,24 @@
+# https://github.com/actions/labeler#common-examples
+# Adapted from https://github.com/rapidsai/cuspatial/blob/main/.github/CODEOWNERS
+# Labels culled from https://github.com/rapidsai/cuspatial/labels
+
+Python:
+  - 'python/**'
+  - 'notebooks/**'
+  
+cpp:
```

Comment:
There is no `cpp` label.

Reply:
do you know what I can replace cpp with?

Reply:
probably fine to leave this. based on Mike's comment in the other PRs, this label will be created if it doesn't already exist.